ex7.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+
+def replace_numbers(array, number):
+    """
+    Replace all numbers in a given array which is equal, less or greater to a given number.
+    :param array: The original array.
+    :param number: The given number.
+    """
+    replacement_number = -1
+    print(np.where(array == number, replacement_number, array))
+    print(np.where(array < number, replacement_number, array))
+    print(np.where(array > number, replacement_number, array))
```

Comment:
You should seperate between logic and output, returning the result insteasd of printing will make this function more useful.